lib/engine/route.rb

```diff
@@ -342,7 +347,12 @@ def subsidy
     end
 
     def revenue_str
-      @revenue_str ||= @game.revenue_str(self)
+      @revenue_str ||=
+        if @route_error
+          visited_stops.map { |s| s.hex.name }.join('-')
```

Comment:
Is there a base game method that does just this part? It'd be better if we weren't duplicating logic.

Reply:
It's close to what the game `revenue_str` method does. I'm going to delete this and just keep the 1822CA fix.